gateway/README.md

```diff
@@ -11,7 +11,9 @@ suspend fun main(args: Array<String>) {
     val gateway = DefaultGateway { // optional builder for custom configuration
         client = HttpClient {
             install(WebSockets)
-            install(JsonFeature)
+            install(ContentNegotiation) {
+                json()
+            }
```

Comment:
I am quite sure this is unnecessary

Reply:
what part specifically? all of it?

Reply:
or do you mean because it is gateway and we don't do http requests?

Reply:
we neither do http requests nor do we use the content negotiation websocket feature